themes/odin-theme/layouts/partials/header.html

```diff
@@ -22,12 +22,32 @@
 
     gtag('config', 'UA-67516878-2');
   </script>
+
+  <!-- Favicons-->
+  <link rel="apple-touch-icon" sizes="57x57" href="/favicons/apple-icon-57x57.png">
+  <link rel="apple-touch-icon" sizes="60x60" href="/favicons/apple-icon-60x60.png">
+  <link rel="apple-touch-icon" sizes="72x72" href="/favicons/apple-icon-72x72.png">
+  <link rel="apple-touch-icon" sizes="76x76" href="/favicons/apple-icon-76x76.png">
+  <link rel="apple-touch-icon" sizes="114x114" href="/favicons/apple-icon-114x114.png">
+  <link rel="apple-touch-icon" sizes="120x120" href="/favicons/apple-icon-120x120.png">
+  <link rel="apple-touch-icon" sizes="144x144" href="/favicons/apple-icon-144x144.png">
+  <link rel="apple-touch-icon" sizes="152x152" href="/favicons/apple-icon-152x152.png">
+  <link rel="apple-touch-icon" sizes="180x180" href="/favicons/apple-icon-180x180.png">
+  <link rel="icon" type="image/png" sizes="192x192"  href="/favicons/android-icon-192x192.png">
+  <link rel="icon" type="image/png" sizes="32x32" href="/favicons/favicon-32x32.png">
+  <link rel="icon" type="image/png" sizes="96x96" href="/favicons/favicon-96x96.png">
+  <link rel="icon" type="image/png" sizes="16x16" href="/favicons/favicon-16x16.png">
+  <link rel="manifest" href="/favicons/manifest.json">
+  <meta name="msapplication-TileColor" content="#ffffff">
+  <meta name="msapplication-TileImage" content="/favicons/ms-icon-144x144.png">
+  <meta name="theme-color" content="#ffffff">
+
 </head>
 <body>
 <div class="wrapper">
 <header>
   <nav>
-    <h1 id="logo"><a href="/"><img src="/images/logo-slim.png" alt="Odin" id="nav-log"></a></h1>
+    <h1 id="logo"><a href="/"><img src="/images/logo-icon.png" alt="Odin" id="nav-log"></a></h1>
```

Comment:
I'm not a big fan using the **O** as the image at the top of the page. Just keep it as the logo-slim.png

Reply:
I think it's cheesy to have the same logo in the upper left and the same logo on the main homepage. It's just duplicated. 

Maybe the new logo can go in the upper left instead?